### ISSUE TITLE
[BUG] Strong name signature not valid for this assembly SkiaSharp.dll

### ISSUE THREAD

**Issue by Inglonias** [mono/SkiaSharp#1805]
**Description**

Attempting to run the setup.exe file generated by my project that uses SkiaSharp results in an error when I try to install.

**Code**

**Expected Behavior**

**Actual Behavior**

```PLATFORM VERSION INFO
	Windows 			: 10.0.19043.0 (Win32NT)
	Common Language Runtime 	: 4.0.30319.42000
	System.Deployment.dll 		: 4.8.4270.0 built by: NET48REL1LAST_C
	clr.dll 			: 4.8.4400.0 built by: NET48REL1LAST_C
	dfdll.dll 			: 4.8.4270.0 built by: NET48REL1LAST_C
	dfshim.dll 			: 10.0.19041.1 (WinBuild.160101.0800)

SOURCES
	Deployment url			: file:///C:/Users/USERNAME/Desktop/VGraph.application
	Application url			: file:///C:/Users/USERNAME/Desktop/Application%20Files/VGraph_1_0_0_0/VGraph.exe.manifest

IDENTITIES
	Deployment Identity		: VGraph.application, Version=1.0.0.0, Culture=neutral, PublicKeyToken=aa96b6441515b6a3, processorArchitecture=msil
	Application Identity		: VGraph.exe, Version=1.0.0.0, Culture=neutral, PublicKeyToken=aa96b6441515b6a3, processorArchitecture=msil, type=win32

APPLICATION SUMMARY
	* Installable application.

ERROR SUMMARY
	Below is a summary of the errors, details of these errors are listed later in the log.
	* Activation of C:\Users\USERNAME\Desktop\VGraph.application resulted in exception. Following failure messages were detected:
		+ Strong name signature not valid for this assembly SkiaSharp.dll.

COMPONENT STORE TRANSACTION FAILURE SUMMARY
	No transaction error was detected.

WARNINGS
	There were no warnings during this operation.

OPERATION PROGRESS STATUS
	* [2021-09-10 5:20:30 PM] : Activation of C:\Users\USERNAME\Desktop\VGraph.application has started.
	* [2021-09-10 5:20:30 PM] : Processing of deployment manifest has successfully completed.
	* [2021-09-10 5:20:30 PM] : Installation of the application has started.
	* [2021-09-10 5:20:30 PM] : Processing of application manifest has successfully completed.
	* [2021-09-10 5:20:32 PM] : Found compatible runtime version 4.0.30319.
	* [2021-09-10 5:20:32 PM] : Request of trust and detection of platform is complete.

ERROR DETAILS
	Following errors were detected during this operation.
	* [2021-09-10 5:20:33 PM] System.Deployment.Application.InvalidDeploymentException (SignatureValidation)
		- Strong name signature not valid for this assembly SkiaSharp.dll.
		- Source: System.Deployment
		- Stack trace:
			at System.Deployment.Application.ComponentVerifier.VerifyStrongNameAssembly(String filePath, AssemblyManifest assemblyManifest)
			at System.Deployment.Application.ComponentVerifier.VerifyComponents()
			at System.Deployment.Application.DownloadManager.DownloadDependencies(SubscriptionState subState, AssemblyManifest deployManifest, AssemblyManifest appManifest, Uri sourceUriBase, String targetDirectory, String group, IDownloadNotification notification, DownloadOptions options)
			at System.Deployment.Application.ApplicationActivator.DownloadApplication(SubscriptionState subState, ActivationDescription actDesc, Int64 transactionId, TempDirectory& downloadTemp)
			at System.Deployment.Application.ApplicationActivator.InstallApplication(SubscriptionState& subState, ActivationDescription actDesc)
			at System.Deployment.Application.ApplicationActivator.PerformDeploymentActivation(Uri activationUri, Boolean isShortcut, String textualSubId, String deploymentProviderUrlFromExtension, BrowserSettings browserSettings, String& errorPageUrl, Uri& deploymentUri)
			at System.Deployment.Application.ApplicationActivator.PerformDeploymentActivationWithRetry(Uri activationUri, Boolean isShortcut, String textualSubId, String deploymentProviderUrlFromExtension, BrowserSettings browserSettings, String& errorPageUrl)
--- End of stack trace from previous location where exception was thrown ---
			at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
			at System.Deployment.Application.ApplicationActivator.PerformDeploymentActivationWithRetry(Uri activationUri, Boolean isShortcut, String textualSubId, String deploymentProviderUrlFromExtension, BrowserSettings browserSettings, String& errorPageUrl)
			at System.Deployment.Application.ApplicationActivator.ActivateDeploymentWorker(Object state)

COMPONENT STORE TRANSACTION DETAILS
	No transaction information is available.

```

**Basic Information**

- Version with issue:  2.80.3
- Last known good version:  Unknown
- IDE:  Visual Studio 2019 Community
- Platform Target Frameworks: Windows
- Target Devices:   Windows PC
  
<details>
  <summary>Detailed IDE/OS information (click to expand)</summary>
  
```

PASTE ANY DETAILED VERSION INFO HERE

```
</details>

**Screenshots**

N/A

**Reproduction Link**
The project I am trying to publish is [hosted here on GitHub](https://github.com/Inglonias/VGraph)

**Comment by Inglonias** [comment]
...Turns out I can just create a release by zipping everything up and uploading it, so this isn't as big a problem as I first thought.

**Comment by mattleibow** [comment]
Duplicate of #1767

TL;DR: There is a preview of 2.80.4 that has been released just for this issue.
